privacyidea/lib/crypto.py

```diff
@@ -307,86 +310,92 @@ def decryptPin(cryptPin):
 
 
 @log_with(log, log_entry=False)
-def encrypt(data, iv, id=0):
-    '''
+def encrypt(data, iv, key_id=0):
+    """
     encrypt a variable from the given input with an initialisation vector
 
-    :param data: buffer, which contains the value
-    :type  data: bytes or str
-    :param iv:   initialisation vector
-    :type  iv:   bytes or str
-    :param id:   contains the key id of the keyset which should be used
-    :type  id:   int
-    :return:     encrypted and hexlified data
+    :param data:   buffer, which contains the value
+    :type  data:   bytes or str
+    :param iv:     initialisation vector
+    :type  iv:     bytes or str
+    :param key_id: contains the key id of the keyset which should be used
+    :type  key_id: int
+    :return: encrypted and hexlified data
     :rtype: str
-
-    '''
+    """
     hsm = get_hsm()
-    ret = hsm.encrypt(to_bytes(data), to_bytes(iv), id)
+    ret = hsm.encrypt(to_bytes(data), to_bytes(iv), key_id)
     return hexlify_and_unicode(ret)
 
 
 @log_with(log, log_exit=False)
-def decrypt(input, iv, id=0):
-    '''
-    decrypt a variable from the given input with an initialiation vector
-
-    :param input: buffer, which contains the crypted value
-    :type  input: bytes or str
-    :param iv:    initialisation vector
-    :type  iv:    bytes or str
-    :param id:    contains the key id of the keyset which should be used
-    :type  id:    int
-    :return:      decrypted buffer
+def decrypt(enc_data, iv, key_id=0):
```

Comment:
should we maybe call the parameters the same in all functions? Above in ``aes_decrypt`` the ciphertext parameter is called *data_input*. Here is is called *enc_data*. (Maybe we should also choose *enc_data* in the above function?)

Reply:
makes sense as well